### ISSUE TITLE
github extension for zed

### ISSUE THREAD

**Issue by UriZafrir** [zed-industries/extensions#623]
### Check for existing issues

- [X] Completed

### Misc notes

Hi, I really like Zed!!
I'm really missing a github extension for it to completely move from vscode.
Any chance to implement it?
thanks!

**Comment by eth0net** [comment]
Might be worth noting the features you want most, or a link to the VS Code extension, for example. 🙂

**Comment by UriZafrir** [comment]
I'm missing the most - just using the source control in vscode is great. being able to create a repo from a folder, being able to commit and push, color highlighting on .gitignore files, being able to see the remote, being able to create a new branch, see which branch I'm at. Being able to authenticate to github and create a private repo, being able to push to private repo from vscode..
maybe this is a duplicate of https://github.com/zed-industries/zed/issues/4367

**Comment by d1y** [comment]
> color highlighting on .gitignore files

The `git firefly` extension support this :)

**Comment by eth0net** [comment]
Yeah a lot of those are git-related and not GitHub specific :) The only GitHub specific thing in that list would be creating a new private GitHub repo. Creating a new repo and pushing etc is all just git, so yeah that would come in nicely with the other issue.

**Comment by Askath** [comment]
Seeing issues and pull requests similar to how IntelliJ from Jetbrains does would also be cool. Although not really a dealbreaker.

**Comment by talha5389-teraception** [comment]
I would love github pull request extension support. That is a major hurdle for me in migrating to zed https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github
